src/main/java/at/hannibal2/skyhanni/data/PurseAPI.kt

```diff
@@ -7,12 +7,16 @@ import at.hannibal2.skyhanni.events.PurseChangeEvent
 import at.hannibal2.skyhanni.utils.NumberUtil.formatNumber
 import at.hannibal2.skyhanni.utils.NumberUtil.milion
 import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
 import net.minecraft.client.Minecraft
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 class PurseAPI {
-    // TODO USE SH-REPO
-    private val pattern = "(Piggy|Purse): §6(?<coins>[\\d,]*).*".toPattern()
+    private val coinsPattern by RepoPattern.pattern(
```

Comment:
I'd respect it if you dont do purse pattern since my scoreboard pr needs it and uhh its very diff